src/test/FuzzerImpl.cpp

```diff
@@ -446,6 +570,7 @@ resetTxInternalState(Application& app)
 {
     resetRandomSeed();
     FuzzUtils::resetSequenceNumber();
+    FuzzUtils::clearStoredLedgerKeys();
```

Comment:
This seems like it is not going to do what you want it to do. After you call this, `numStoredLedgerKeys = 0`. But that means you are going to lose all the keys that were created during the set-up phase. Am I missing something?

Reply:
No, I was missing something!  I meant to change that method to something that reverts to post-setup rather than completely-uninitialized state, but I forgot.  It should now do so (in a different way, since the data structures have changed).